config/reader.go

```diff
@@ -49,31 +58,154 @@ func getAgentInfo() (*apppb.AgentInfo, error) {
 	}, nil
 }
 
-var viamDotDir = filepath.Join(os.Getenv("HOME"), ".viam")
+// takes a config ID, tries to read it and returns the file in bytes.
+func readFromCache(id string) (*Config, error) {
+	filepath := path.Clean(getCloudCacheFilePath(id))
+	// first check the path exists
+	if _, err := os.Stat(filepath); err != nil {
```

Comment:
i replaced the os.reader function with Readfile which doesn't return the same error if the file doesn't exist. os.stat() does do that though which now firsts checks if the filepath exists before trying to read the file.

Reply:
thanks ! I was trying to figure out what the return type of this error was but couldn't so thank you!